views/index.blade.php

```diff
@@ -28,14 +28,37 @@
 
 <div id="graphiql">Loading...</div>
 <script src="{{ \MLL\GraphiQL\DownloadAssetsCommand::jsPath() }}"></script>
+<script src="{{ \MLL\GraphiQL\DownloadAssetsCommand::explorerPluginPath() }}"></script>
 <script>
+    var fetcher = GraphiQL.createFetcher({
+        url: '{{ url(config('graphiql.endpoint')) }}',
+        subscriptionUrl: '{{ config('graphiql.subscription-endpoint') }}',
+    });
+
+    function GraphiQLWithExplorer() {
+        var [query, setQuery] = React.useState(
+            '',
+        );
+
+        var explorerPlugin = GraphiQLPluginExplorer.useExplorerPlugin({
+            query: query,
+            onEdit: setQuery,
+        });
+
+        return React.createElement(GraphiQL, {
+            fetcher: fetcher,
+            defaultEditorToolsVisibility: true,
+            plugins: [explorerPlugin],
+            query: query,
+            onEditQuery: setQuery,
+            headerEditorEnabled: true,
+            shouldPersistHeaders: true,
+            defaultHeaders: '{ "Authorization": "Bearer <ENTER YOUR API KEY>" }'
```

Comment:
Those settings seem orthogonal to adding the plugin, please change only what is strictly necessary